From 74a265f9f04f7f5ff96800f5f0fb3ad31faf5427 Mon Sep 17 00:00:00 2001
From: Marius Bakke <mbakke@fastmail.com>
Date: Sun, 22 Oct 2017 03:27:51 +0200
Subject: [PATCH 2/9] helper: Declare dependency on Qt5::Qml.

Authrequest.h includes <QtQml>, so make sure it's found on systems
where it exists in a different directory from the rest of Qt.
---
 src/helper/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/helper/CMakeLists.txt b/src/helper/CMakeLists.txt
index 6e13653..ebf4a6e 100644
--- a/src/helper/CMakeLists.txt
+++ b/src/helper/CMakeLists.txt
@@ -31,7 +31,7 @@ else()
 endif()
 
 add_executable(sddm-helper ${HELPER_SOURCES})
-target_link_libraries(sddm-helper Qt5::Network Qt5::DBus)
+target_link_libraries(sddm-helper Qt5::Network Qt5::DBus Qt5::Qml)
 if(PAM_FOUND)
     target_link_libraries(sddm-helper ${PAM_LIBRARIES})
 else()
-- 
2.8.3

